tests/test_cli.py

```diff
@@ -74,7 +74,7 @@ def check_csvs(
 class TestCLI(FileBasedTesting):
 
     test_data_dir = os.path.join(os.path.dirname(__file__), 'data')
-
+    # TODO: need to fix in scancode
```

Comment:
`cli/scan_1_file_moved_new.json` is having `fingerprint` attribute so this test will fail as VirtualCodebase do not support `fingerprint` attribute